packages/app/__test__/Titlebar.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+
+import { render, screen } from '@testing-library/react';
+import Titlebar from '../src/components/menu/titlebar/Titlebar.jsx';
+import rootReducer from '../src/store';
+
+describe('<Titlebar>', () => {
+  it('should render loading stage from prop', () => {
+    const store = createStore(rootReducer);
+    const Wrapper = ({ children }) => <Provider store={store}>{children}</Provider>;
+    render(<Titlebar loadingStage='test'/>, { wrapper: Wrapper });
+    screen.getByText('test');
```

Comment:
Shouldn't we have an assertion here? As in the testing-library docs:
```
  // .toBeInTheDocument() is an assertion that comes from jest-dom
  // otherwise you could use .toBeDefined()
  expect(screen.getByText(testMessage)).toBeInTheDocument()
```
It will crash, of course, but "expect"-ations are much clearer.